src/main/java/net/dv8tion/jda/api/events/guild/invite/GuildInviteCreateEvent.java

```diff
@@ -30,7 +30,7 @@
  * <h2>Requirements</h2>
  *
  * <p>This event requires the {@link net.dv8tion.jda.api.requests.GatewayIntent#GUILD_INVITES GUILD_INVITES} intent to be enabled.
- * <br>Also, this event will only fire for invites created in channels where you can {@link net.dv8tion.jda.api.Permission#MANAGE_CHANNEL MANAGE_CHANNEL}
+ * <br>Also, this event will only fire for invites created in channels where you can {@link net.dv8tion.jda.api.Permission#MANAGE_CHANNEL MANAGE_CHANNEL}1  1.
```

Comment:
```suggestion
 * <br>Also, this event will only fire for invites created in channels where you can {@link net.dv8tion.jda.api.Permission#MANAGE_CHANNEL MANAGE_CHANNEL}.
```

Reply:
You're really good at finding these little things really quickly, lmao